versions.props

```diff
@@ -20,8 +20,10 @@ commons-codec:commons-codec=1.15
 commons-collections:commons-collections=3.2.2
 commons-io:commons-io=2.11.0
 io.dropwizard.metrics:*=4.2.12
+io.grpc:grpc-*=1.50.2
 io.jaegertracing:*=1.8.1
 io.netty:*=4.1.82.Final
+io.opentelemetry:opentelemetry-bom-alpha=1.21.0-alpha
```

Comment:
What version do we need to get to to NOT use an alpha version?

Reply:
Well, it is a bit misleading here, since this BOM depends also on the `1.21.0` bom. Most APIs are stable, but the Log SDK is still in Alpha. See https://github.com/open-telemetry/opentelemetry-java#sdk for which are stable and alpha.
The reason we include the alpha bom is that we use the `opentelemetry-sdk-extension-autoconfigure` extension which is still alpha. I guess when it graduates, we can move to only using the stable BOM. In other words, gradle will pick the stable version if it exists, else fallback to alpha, as can be seen from the `lib/` listing I posted above.

Reply:
yea just hoping we can move from "alpha" when possible.